Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -0,0 +1,98 @@
+        {
+            // ToDo: implement.
+        }
+
+        /// <summary>
+        /// Remove last element in array.
+        /// </summary>
+        /// <param name="array">Input array.</param>
+        /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
+        public static int[] RemoveLast(int[] array)
+        {
+            // ToDo: implement.
+            return array;
+        }
+
+        /// <summary>
+        /// Remove first element in array.
+        /// </summary>
+        /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
+        public static int[] RemoveFirst(int[] array)
+        {
+            // ToDo: implement.
+            return array;
+        }
+
+        /// <summary>
+        /// Removes array element at given index.
+        /// </summary>
+        /// <param name="array">Input array.</param>
+        /// <param name="index">Index at which array element should be removed.</param>
+        /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
+        public static int[] RemoveAt(int[] array, int index)
+        {
+            // ToDo: implement.
+            return array;
+        }
+
+        /// <summary>
+        /// Inserts a new array element at the start.
+        /// </summary>
+        /// <param name="array">Input array.</param>
+        /// <param name="number">Number to be added.</param>
+        /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
+        public static int[] InsertFirst(int[] array, int number)
+        {
+            // ToDo: implement.
+            return array;
+        }
+
+        /// <summary>
+        /// Inserts a new array element at the end.
+        /// </summary>
+        /// <param name="array">Input array.</param>
+        /// <param name="number">Number to be added.</param>
+        /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
+        public static int[] InsertLast(int[] array, int number)
+        {
+            // ToDo: implement.
+            return array;
+        }
+
+        /// <summary>
+        /// Inserts a new array element at the specified index.
+        /// </summary>
+        /// <param name="array">Input array.</param>
+        /// <param name="number">Number to be added.</param>
+        /// <param name="index">Index at which array element should be added.</param>
+        /// <returns>A new array with element inserted at a given index. If an array is empty or null, returns new array with number in it.</returns>
+        public static int[] InsertAt(int[] array, int number, int index)
+        {
+            // ToDo: implement.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886045)